d2l-sort-by.html

```diff
@@ -0,0 +1,97 @@
+<link rel="import" href="../polymer/polymer-element.html">
+<link rel="import" href="../d2l-dropdown/d2l-dropdown-menu.html">
+<link rel="import" href="../d2l-dropdown/d2l-dropdown-button-subtle.html">
+<link rel="import" href="../d2l-menu/d2l-menu.html">
+
+<dom-module id="d2l-sort-by">
+	<template strip-whitespace>
+		<style>
+			:host {
+				display: inline-block;
+			}
+		</style>
+		<d2l-dropdown-button-subtle text="[[_buttonText]]">
+			<d2l-dropdown-menu no-padding no-auto-fit id="d2l-sort-by-menu">
```

Comment:
Note: `no-padding` should be the default for `d2l-dropdown-menu`, so it should be possible to omit this here.  (all our our menus should go edge-to-edge in dropdowns)